core/ledger/kvledger/txmgmt/privacyenabledstate/test_exports.go

```diff
@@ -118,11 +120,12 @@ func (env *CouchDBCommonStorageTestEnv) setupCouch() string {
 
 // Init implements corresponding function from interface TestEnv
 func (env *CouchDBCommonStorageTestEnv) Init(t testing.TB) {
-	redoPath, err := ioutil.TempDir("", "pestate")
+	redoPath, err := ioutil.TempDir("", fmt.Sprintf("test-env-%s-", t.Name()))
```

Comment:
```suggestion
	redoPath, err := ioutil.TempDir("", fmt.Sprintf("bookkeeping-%s-", t.Name()))
```

Reply:
https://github.com/hyperledger/fabric/pull/351/files#diff-4bd6ce0c76aa7b9fb651a7bdb242dd0eL36-R38
I think you mean those lines? for bookkeeping? in fact current file is used for couched or leveldb, if we want to naming it... "leveldbtest-env-%s" ?

Reply:
resolve as out of date